src/notification.cpp

```diff
@@ -316,7 +316,11 @@ void NotificationTimer::pause()
         return;
 
     stop();
-    m_intervalMsec = m_startTime.msecsTo(QDateTime());
+    qint64 elapsed = m_startTime.msecsTo(QDateTime::currentDateTime());
+    if (elapsed < m_intervalMsec / 2)
```

Comment:
I don't get this...
I see that that there was a bug before (`QDateTime()` is a invalid timestamp and `msecsTo` gives 0), but what is this condition about?
Why not just:
```c++
m_intervalMsec -= m_startTime.msecsTo(QDateTime::currentDateTime());
```